Ixx: 0.00264
Iyy: 0.00264
Izz: 0.00496
mass: 0.5
g: 9.81
thrust_coefficient: 0.0000000555
prop_radius: 0.099
arm_length: 0.17
motor_time_constant: 0.05
min_rpm: 1500
max_rpm: 7500

